examples/tv-app/tv-common/src/AppTv.cpp

```diff
@@ -607,6 +643,10 @@ CHIP_ERROR AppTvInit()
 #if CHIP_DEVICE_CONFIG_APP_PLATFORM_ENABLED
     ContentAppPlatform::GetInstance().SetupAppPlatform();
     ContentAppPlatform::GetInstance().SetContentAppFactory(&gFactory);
+    gFactory.AddContentApp((uint16_t)1, (uint16_t)11);
+    // gFactory.AddContentApp((uint16_t)65521, (uint16_t)32768);
```

Comment:
Maybe keep this line but provide a way to uninstall it from the AppTvShellCommands for testing?

Reply:
Re-enabled. I've added also AppTvShellCommands for install uninstall.